api/openshift_info.py

```diff
@@ -0,0 +1,97 @@
+import os
+from kubernetes import client, config
+from openshift.dynamic import DynamicClient
+from helper import helper
+
+
+class ServiceClass(object):
+    def __init__(self):
+    # to run inside the openshift cluster
+        if "OPENSHIFT_BUILD_NAME" in os.environ:
+            service_account_path = '/var/run/secrets/kubernetes.io/serviceaccount'
```

Comment:
Don't use Magic values. Always, assign values to variables and use those variables in the code.

Reply:
I missed out with this one, I have this code such that it accepts code from environment variables